apps/extension/src/contents/gmail.tsx

```diff
@@ -1,135 +1,134 @@
-
-    shadowRoot.appendChild(customPortal)
-  }
-  // after hi-jack, return the anchor
-  return gmailSendButton
-}
-
-// this makes plasmo inject our style w/ tw into the shadow dom
-// note (richard): necessary to be able to use tw on the below react component
-export const getStyle = () => {
-  const style = document.createElement("style")
-  style.textContent = tailwindCss
-  return style
-}
-
-const Modal: React.FC<{ children: React.ReactNode }> = ({ children }) => {
-  const [open, setOpen] = useState(false) // to pass to radix
-  const portalContainer = document // our injected portal (next to body)
-    .getElementById(SHADOW_HOST_ID)
-    ?.shadowRoot?.getElementById(PORTAL_ID)
-
-  return (
-    <Dialog.Root open={open} onOpenChange={setOpen}>
-      <Dialog.Trigger className="bg-white rounded px-1.5 py-1.5  hover:bg-gray-50 focus:outline-none ">
-        <img
-          className="w-6"
-          src="https://www.agreeto.app/%2Flogo.png"
-          alt="AgreeTo Logo"
-        />
-      </Dialog.Trigger>
-      <Dialog.Portal container={portalContainer}>
-        <Dialog.Overlay className="fixed w-screen h-screen bg-black bg-opacity-50 pointer-events-auto z-[2147483646]">
-          <Dialog.Content
-            id="agreeto-app"
-            className="fixed -translate-x-1/2 -translate-y-1/2 bg-white shadow-sm top-1/2 left-1/2 shadow-transparent l-1/2 z-[2147483646] w-[800] h-[600]">
-            {children}
-          </Dialog.Content>
-        </Dialog.Overlay>
-      </Dialog.Portal>
-    </Dialog.Root>
-  )
-}
-
-// The icon button that we inject next to our anchor (send btn)
-const GmailContent = () => {
-  // note (richard): no idea why this is necessary, but encountering this error:
-  // https://github.com/theKashey/react-remove-scroll-bar/blob/3dd80e28c92b8aac025e41f4258ff926cdc88af9/src/utils.ts#L22
-
-  if (process.env.NODE_ENV !== "production")
-    document.body.style.cssText = "overflow-y: auto!important;"
-
-  const { isAuthed, isLoading } = useIsAuthed()
-
-  const [location] = React.useState(
-    () => new ReactLocation(reactLocationOptions)
-  )
-
-  return (
-    <div className="pl-1">
-      <Modal>
-        <Router location={location} routes={getRoutes()}>
-          <div className="h-[600] w-[800]">
-            {isLoading ? (
-              <div className="h-full w-full grid place-content-center">
-                <div className="h-12 w-12 rounded-full border-2 animate-pulse"></div>
-              </div>
-            ) : isAuthed ? (
-              // Render app layout if user is authed
-              <Layout>
-                <Outlet />
-              </Layout>
-            ) : (
-              // Render sign in page if user is not authed
-              <SignIn />
-            )}
-          </div>
-        </Router>
-      </Modal>
-    </div>
-  )
-}
-
-const Gmail = () => {
-  return (
-    <TRPCProvider>
-      <GmailContent />
-    </TRPCProvider>
-  )
-}
-
-export default Gmail
+export {};
```

Comment:
idk why but im getting an error here about the script not being utf-8 encoded. any ideas?

Reply:
No, not sure. But we can ask plasmo tomorrow why that is.

Reply:
I don't know either